### DISCUSSION TITLE
Any plan for a VST plugin to run SOUL patches?

### DISCUSSION THREAD

**Discussion by giohappy** [soul-lang/SOUL#66] [Q&A] [answered]
I've read that SOUL patches can be run in Waveforms,and that you can generate C++ (and JUCE?) code from which a VST could be built, but is there any plan to offer a VST plugin where SOUL patches can be run and JIT compiled on the fly?

**Comment by julianstorer** [comment, marked as answer]
There is one - code is here:
https://github.com/soul-lang/SOUL/blob/master/tools/plugin/Patch_Plugin_README.md

**Comment by giohappy** [comment]
A part from an issue with VS 2019 and pdb files I could build the plugin, but now I'm hitting my head against ths "Coulnd't fin or load SOUL_PatchLoader.dll" message.

![image](https://user-images.githubusercontent.com/571129/117330227-c09ff280-ae95-11eb-9053-752363ba8aa9.png)

I've tried putting in the same folder beside the vst3 plugin, under the (odd) AppData/SOUL folder, but noway... the message still apears.

**Reply by LibroLibri** [reply]
I have the same problem. With 1.0.82 I was able to get the standalone version working, but it still happens with the VST3 build.
